modules/installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc

```diff
@@ -57,82 +57,149 @@ podman login -u kubeadmin -p $(oc whoami -t) $REGISTRY_HOST
 podman login registry.redhat.io
 ----
 +
-For more information about registry authentication, see https://access.redhat.com/RegistryAuthentication[{company-name} Container Registry Authentication].
-
-. Pull {product-short} and PostgreSQL images from https://catalog.redhat.com/software/containers/search[{company-name} Image registry] to your workstation, by running the following commands:
+. Create an `ImageSetConfiguration` file to specify the resources that you want to mirror. For example:
++
+[source,terminal]
+----
+apiVersion: mirror.openshift.io/v1alpha2
+kind: ImageSetConfiguration
+storageConfig:
```

Comment:
@Fortune-Ndlovu I think this `storageConfig` section is not needed and can be bypassed if we are mirroring directly to the target mirror registry.
`oc-mirror` creates a folder named `oc-mirror-workspace` in the current directory.